### PR TITLE
Get data from google doc for Announcements

### LINKED ISSUE #1327: Automating Announcement Section
Hello - The Node.js Foundation is now issuing all of its announcements on Medium. Given this, I thought we should update our announcement section to be more like the "In the News" section. Essentially, I just put the title and link into a google document and the "In the News" section is automatically updated.

Is this group okay with this and if so, would anyone know how to do this? I'm sure it's fairly simple, I just don't know how to go about and create it.

### PR DESCRIPTION
**Pull request by pierreneter**
Enable show data from Google Doc.
close #1327

### CODE REVIEW

File: layouts/announcements.hbs [hunk 1]
```diff
@@ -11,25 +11,72 @@
...
           </ul>
-
         </div>
       </article>
-
     </div>
   </div>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/tabletop.js/1.4.2/tabletop.min.js"></script>
+  <script type="text/javascript">
+  var public_spreadsheet_url = 'https://docs.google.com/spreadsheets/d/1xwQ6ciZLADL43My_XBIWLVRuDMl8cx_omYd1FRkBZ58/pubhtml';
```
Comment (fhemberger) [inline layouts/announcements.hbs]: We use the standard.js style across the nodejs.org website, please make sure client side scripts follow the same style.

File: layouts/in-the-news.hbs [hunk 2]
```diff
@@ -25,16 +26,16 @@
       window.onload = function () {
```
Comment (fhemberger) [inline layouts/in-the-news.hbs]: We use the standard.js style across the nodejs.org website, please make sure client side scripts follow the same style.

**Review by fhemberger — COMMENTED** [review]
Two little nits, otherwise LGTM. 👍 
Sorry for the delayed review!

File: layouts/announcements.hbs (hunk 1 above)
Comment (pierreneter) [reply layouts/announcements.hbs]: It was copied from [layouts/in-the-news.hbs#L22](https://github.com/nodejs/nodejs.org/master/layouts/in-the-news.hbs#L22) to use layout as news sections. If something goes wrong I'll fix it on both pages.

Comment (pierreneter) [reply layouts/announcements.hbs]: It was copied from [layouts/in-the-news.hbs#L22](https://github.com/nodejs/nodejs.org/blob/master/layouts/in-the-news.hbs#L22) to use layout as news sections. If something goes wrong I'll fix it on both pages.

Comment (fhemberger) [reply layouts/announcements.hbs]: @pierreneter The code below now has semicolons at the end, for example, which is not the case for `in-the-news.hbs` but had other issues fixed in #1352.

Could you please fix that so it validates against standard style?
See https://standardjs.com/demo.html for a quick check.

**Review by fhemberger — CHANGES_REQUESTED** [review]

**Comment by lpinca** [comment]
Can you also please remove yarn.lock? We already have package-lock.json or do we want to keep both?

File: layouts/announcements.hbs (hunk 1 above)
Comment (pierreneter) [reply layouts/announcements.hbs]: Yes. I will push a commit tonight

**Comment by pierreneter** [comment]
@fhemberger @lpinca all fix, thanks.

**Comment by pierreneter** [comment]
@fhemberger @lpinca all fixed, thanks.

**Review by pierreneter — COMMENTED** [review]
Sorry for my shortcoming, thank you.

**Review by fhemberger — APPROVED** [review]

**Comment by fhemberger** [comment]
Great, thank you!

**Pull request merged**